physionet-django/project/admin.py

```diff
@@ -3,7 +3,7 @@
 from django.contrib import admin
 from django.db.models import CharField, TextField
 from django.forms import TextInput, Textarea
-from background_task.models_completed import CompletedTask
+#from background_task.models_completed import CompletedTask
```

Comment:
Why did you remove it?

Reply:
Ok, uncommented again. I had too new version which didn't have such module `models_completed` and I was wondering why it's not working for me. So I figured out in `django-background-tasks` 1.2.0 there IS `models_completed` module and in 1.2.6 there IS NOT which is a breaking change in a x.y.Z version change, not the best thing to do.